lib/package-manager/install.js

```diff
@@ -4,25 +4,33 @@ const path = require('path');
 const stripAnsi = require('strip-ansi');
```

Comment:
If you're already renaming, maybe rename this file to `installer`. As first glance I thought it's intended to install the package-manager inself.

Reply:
Valid point.